canonical-web-and-design/jaas-dashboard
# Convert entity panels to pages

## Done

- Converted units / machines / app panels to pages
- Redirect to 404 if trying to access a model via refresh or direct URL that is not accessible
- Update and simplify `<ChipGroup />`
- Uncouple entity metadata into a separate component from `<InfoPanel>`
- Fix bouncing sidebar
- Updated Breadcrumbs to be dynamic to different entity types

## QA

- Run the app using `yarn start` or demo link
- Click around and make sure everything works as expected

Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1624,
Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1621,
Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1623,
Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1625,
Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1628,
Fixes https://github.com/canonical-web-and-design/jaas-dashboard/issues/885
